docs/configuration/configuration.md

```diff
@@ -134,6 +137,8 @@ There are three different types of logging: standard, authentication, and HTTP r
 
 Each type of logging has their own configurable format and variables. By default these formats are similar to the Apache Combined Log.
 
+Logging of requests to the `/ping` endpoint can be disabled with `-silence-ping-logging` reducing log volume. This flag sets the `-exclude-logging-path` value to the `-ping-path` and takes precedence over any other value `-exclude-logging-path` may have been set to directly.
```

Comment:
Perhaps the values should be merged rather than just having the `ping` disabling everything?

Reply:
Sure, especially if we do a list of excluded paths. This felt a little incomplete when writing. Will amend.

Reply:
Thanks for patience, been a busy couple of weeks. Updated now.
`options.go` creates a slice of paths based on the `silence-ping-logging` bool and `exclude-logging-paths` csv string list, then passes this slice to `logger.go`
`logger.go` converts the slice to a `map[string]struct{}` for quick path lookup.